src/components/tableDrawing/RowAdditionAndFunctionField.jsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import { useDispatch, useSelector } from "react-redux";
+import AddIcon from "@mui/icons-material/Add";
+import PropTypes from "prop-types";
+import ItemInfoPopover from "./ItemInfoPopover";
+import { addNewFieldUnderSelectedRow } from "../../features/tableDataInfo/tableDataInfoSlice";
+
+export default function RowAdditionAndFunctionField({
+  eventIndex,
+  setEvnetIndex,
+}) {
+  const dispatch = useDispatch();
+  const { data } = useSelector((store) => store.tableDataInfo);
+
+  return (
+    <>
+      {Array.from(Array(data[0].data.length).keys()).map((index) => (
```

Comment:
Array(data[0].data.length).keys() 
what is this?

Reply:
I draw my icon through this array